cmd/starsd/cmd/genesis.go

```diff
@@ -553,6 +554,6 @@ func LocalnetGenesisParams() GenesisParams {
 	genParams.GovParams.TallyParams.Quorum = sdk.MustNewDecFromStr("0.1")    // 10%
 	genParams.GovParams.TallyParams.Threshold = sdk.MustNewDecFromStr("0.5") // 50%
 	genParams.GovParams.VotingParams.VotingPeriod = time.Minute * 1          // 5 min
```

Comment:
I this right?

Reply:
This is is the `prep-genesis` command for bootstrapping new networks only, does not apply to mainnet